### DISCUSSION TITLE
反応しないキーがある

### DISCUSSION THREAD

**Discussion by Rindrics** [Yowkees/keyball#239] [Q&A]
素晴らしい製品の開発、ありがとうございます。

ビルドガイドに従って組み立てたのですが、下の写真で赤丸をつけたキーのみ動作させられていません（ピンセットによるショートでも反応なし）。

![IMG_1985](https://github.com/Yowkees/keyball/assets/14845847/33087b35-f78c-48c9-95d1-47604d2d8555)

恐れ入りますが、このキーに対応するダイオードの場所など、確認すべきポイントを教えていただけないでしょうか？

よろしくお願いいたします :bow:

**Comment by Yowkees** [comment]
![44keyfix](https://github.com/Yowkees/keyball/assets/84209772/b92db837-6b56-41f7-a28a-9512dc891cd0)
ご連絡が遅くなり申し訳ありませんでした。
該当キーのダイオードは添付写真の青丸部のものになります。
ピンセットでもダメだったとのことですが、念のため、キーソケットの交換も試していただければ幸いです。
以上お手数ですがご対応お願いいたします。

**Reply by Rindrics** [reply]
ご回答いただき、ありがとうございます。
ご教示いただいたダイオードを新しいものに交換しました。
またキーソケットについては、赤丸で囲ったものを取り外し、キーソケットがない状態でのピンセットショートを試みましたが、残念ながらこの時点で反応しないことがわかりました。この場合、新しいキーソケットをつけても効果がないという認識なのですが、これは合っているでしょうか？

**Comment by Rindrics** [comment]
もし青丸のダイオード以外に確認すべき箇所がありましたら、恐れ入りますが教えていただけないでしょうか？
もし確認できる箇所がなくなった場合、正常に動作させるためには基盤ごと交換する必要がありますか？

**Comment by Yowkees** [comment]
![KB44KeyFix](https://github.com/Yowkees/keyball/assets/84209772/8a7701ba-9623-4272-9146-ae37c8f287f8)
キーソケットもしくはダイオードの取外し時にパッドが引っ張られて線が断線することがありますので、それが原因かも知れません。
画像の同じ色同士の点が導通しているかテスター等で確認いただけますでしょうか？

導通していないものがあれば、エナメル線かゴム被覆導線で接続していただければ、キーを復活できるはずです。
お手数ですがご対応お願いいたします。

**Reply by Rindrics** [reply]
アドバイスいただいた通り、エナメル線を使った対処で無事復活させることができました。ありがとうございます 🙇

**Comment by Rindrics** [comment]
本件、解決したので close させていただきます。ありがとうございます 🙏

**Reply by Rindrics** [reply]
（just idea ですが、Discussion に問題の箇所を示すラベル（例えば `key:17` `side:ball` とか）をつけると、検索性が向上して、重複した質問が少し減るのではないかな？と思いました 😄 @Yowkees ）
